### PR TITLE
added city output to display

### PR DESCRIPTION
**Pull request by printminion**
fixed broken ° symbol for temperature - possible encoding problem
added city output to display - wanted to see the weather location
removed unused imports

### CODE REVIEW

**Comment by metawatchclient** [comment]
Hi russenreaktor,

Meta Watch requires everyone who wants to commit into MWM-for-Android to sign the CLA(Contributor License Agreement).

You can find more information in the README of this project.

To review and sign the agreement:
http://www.metawatch.org/developers/cla.html

Thanks for your support for Meta Watch Community.

BR, 

Siqi
siqi@metawatch.org

**Comment by printminion** [comment]
@metawatchclient: done

File: src/org/metawatch/manager/Monitors.java [hunk 1]
```diff
@@ -166,20 +163,22 @@ public static void updateWeatherData(Context context) {
...
+			WeatherData.city = Preferences.weatherCity;
+			
 			String cond = wcc.getCondition();
 			String temp;
 			if (Preferences.weatherCelsius)
 			{
 				WeatherData.tempHigh = "High " + Integer.toString(wfc.getTempMaxCelsius());
 				WeatherData.tempLow = "Low " + Integer.toString(wfc.getTempMinCelsius());
-				temp = Integer.toString(wcc.getTempCelcius()) + "°C";
+				temp = Integer.toString(wcc.getTempCelcius()) + "CÂ°";
```
Comment (metawatchclient) [inline src/org/metawatch/manager/Monitors.java:175]: The encoding issue can be fixed by:
temp = Integer.toString(wcc.getTempCelcius()) + (char) 0x00B0 + "C";

File: src/org/metawatch/manager/Monitors.java [hunk 2]
```diff
@@ -166,20 +163,22 @@ public static void updateWeatherData(Context context) {
...
 			}
 			else 
 			{
 				WeatherData.tempHigh = "High " + Integer.toString(WeatherUtils.celsiusToFahrenheit(wfc.getTempMaxCelsius()));
 				WeatherData.tempLow = "Low " + Integer.toString(WeatherUtils.celsiusToFahrenheit(wfc.getTempMinCelsius()));
-				temp = Integer.toString(wcc.getTempFahrenheit()) + "°F";
+				temp = Integer.toString(wcc.getTempFahrenheit()) + "F";
```
Comment (metawatchclient) [inline src/org/metawatch/manager/Monitors.java:181]: The encoding issue can be fixed by:
temp = Integer.toString(wcc.getTempCelcius()) + (char) 0x00B0 + "F";

File: src/org/metawatch/manager/MetaWatch.java [hunk 3]
```diff
@@ -124,7 +125,7 @@ void showAbout() {
 		String html = "<html><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\" /><title>About</title></head><body>" + 
 						"<h1>MetaWatch</h1>" +
 						"<p>Version " + Utils.getVersion(this) + ".</p>" +
-						"<p>© Copyright 2011 Meta Watch Ltd.</p>" +
+						"<p>ï¿½ Copyright 2011 Meta Watch Ltd.</p>" +
```
Comment (metawatchclient) [inline src/org/metawatch/manager/MetaWatch.java:128]: You can use either <code>&amp;#169;</code> or <code>&amp;copy;</code> for copyright sign in HTML representation.

**Comment by metawatchclient** [comment]
Hi russenreaktor,

Thanks for signing the CLA. Your code will be merged soon. I will make a commit to fix the encoding issue mentioned above, so you don't need to do anything.

BR, 

Siqi

**Pull request merged**